packages/cli/src/oclif/commands/analytics.js

```diff
@@ -9,6 +9,10 @@ const {
 const colors = require('colors/safe');
 
 class AnalyticsCommand extends BaseCommand {
+  requiresCore() {
+    return false;
+  }
+
```

Comment:
Initially I tried declaring it in a constructor.
```  
class BuildCommand extends BaseCommand {
  constructor() {
    super();
    this.requiresCore = true;
  }
// ...
```  

But I kept getting this error, which seemed to be something with oclif.  
```  
TypeError: Cannot read properties of undefined (reading 'scopedEnvVarKey')
    at BuildCommand._run (~/z/zapier-repos/zapier-platform/packages/cli/node_modules/@oclif/command/lib/command.js:41:44)
    at Command.run (~/z/zapier-repos/zapier-platform/packages/cli/node_modules/@oclif/command/lib/command.js:162:16)
    at async Config.runCommand (~/z/zapier-repos/zapier-platform/packages/cli/node_modules/@oclif/config/lib/config.js:173:24)
    at async Main.run (~/z/zapier-repos/zapier-platform/packages/cli/node_modules/@oclif/command/lib/main.js:27:9)
    at async Main._run (~/z/zapier-repos/zapier-platform/packages/cli/node_modules/@oclif/command/lib/command.js:43:20)  
```  

If you've got any insight happy to hear what might be causing it. I didn't see anything in Oclif's docs and couldn't find anything relevant on Google.